obi_generative_gui/security-groups.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_security_group" "acc_sg" {
+  vpc_id = var.vpc_id
+
+  name        = "main_obi_generative_gui_sg"
+  description = "main security group for obi-generative-gui resources"
+}
+
+resource "aws_vpc_security_group_ingress_rule" "main_subnet_ingress" {
```

Comment:
Do we still need this?

Reply:
Both aws_vpc_security_group_egress_rule and aws_vpc_security_group_ingress_rule are defined in backend.tf.
Would it be safe to remove the full security-groups.tf file?

Reply:
From a quick grep 'main_subnet_ingress' seems to only be used by the SG `acc_sg`. We can probably delete that SG and all the rules assigned to it. 

I think `obi_generative_gui_ecs_task` and `obi_one_ecs_task` SGs are enough (one per service/task).
Unless there is something else like EFS, DBs, etc. But that can be added later